client/src/components/annoDialog.js

```diff
@@ -42,7 +42,7 @@ class AnnoDialog extends React.PureComponent {
               <p
                 style={{
                   marginTop: 7,
-                  visibility: validationError ? "visible" : "hidden",
```

Comment:
validation error doesn't necessarily mean we'll be showing a message

Reply:
Maybe add this comment in the code just in case someone changes it back to `validationError` lol